script/model/mini_trainer.py

```diff
@@ -145,15 +162,16 @@ def train(self):
                          help='Prediction results of the mini models')
     aparser.add_argument('--save_path', default='trained_nogen_model', help='Path to save the mini models')
     aparser.add_argument('--ml_models', nargs='*', type=str,
-                         default=["lr", "rf", "nn", 'huber', 'svr', 'kr', 'gbm'],
+                         default=["lr", "rf", "gbm"],
                          help='ML models for the mini trainer to evaluate')
     aparser.add_argument('--test_ratio', type=float, default=0.2, help='Test data split ratio')
+    aparser.add_argument('--trim', default=0.2, type=float, help='% of values to remove from both top and bottom')
```

Comment:
I don't think we need this trim thing anymore, right?

Reply:
I kept the trim since the script doesn't default to medians for reducing to a single data point. The trim is supposed to be the x% trim.

Reply:
Got it!